R/mod-synapse-oauth.R

```diff
@@ -18,6 +18,10 @@ authorization_url <- NULL
 #' @param pkgname default R .onLoad() parameter
 .onLoad <- function(libname, pkgname) {
   synapse <<- reticulate::import("synapseclient", delay_load = TRUE)
+  #automatically check if the synapseclient version is larger than 2.3.1,
+  #otherwise, print the error message in log file
```

Comment:
Did you specify an error message or is there a default one that gets logged?

Reply:
In the log file, it shows the App execution halted because it did not pass the line 24 in the code.